Stable_Diffusion_WebUi_Altryne.ipynb

```diff
@@ -48,111 +13,102 @@
         "---\n",
         "\n",
         "If this colab helped you, support me on ko-fi and don't forget to subscribe to my awesome list https://github.com/altryne/awesome-ai-art-image-synthesis\n"
-      ],
-      "metadata": {
-        "id": "jnXUwtACCHOy"
-      }
+      ]
     },
     {
       "cell_type": "markdown",
-      "source": [
-        "## 1 -  Setup stage"
-      ],
       "metadata": {
         "id": "4eodrBxeCgOW"
-      }
+      },
+      "source": [
+        "## 1 -  Setup stage"
+      ]
     },
     {
       "cell_type": "code",
-      "execution_count": 1,
+      "execution_count": null,
       "metadata": {
-        "id": "cAsdhafsBvEr",
-        "outputId": "e8510d15-10b1-4b2e-89bc-e0891b375205",
-        "colab": {
-          "base_uri": "https://localhost:8080/"
-        }
+        "id": "cAsdhafsBvEr"
       },
-      "outputs": [
-        {
-          "output_type": "stream",
-          "name": "stdout",
-          "text": [
-            "GPU 0: Tesla P100-PCIE-16GB (UUID: GPU-29dd0cba-97ef-7721-4c2e-e5406353cdaf)\n"
-          ]
-        }
-      ],
+      "outputs": [],
       "source": [
         "!nvidia-smi -L"
       ]
     },
     {
       "cell_type": "markdown",
+      "metadata": {
+        "id": "RXasLtrYC9uj"
+      },
       "source": [
         "### 1.1 Download repo and install\n",
         "\n",
-        "Clone git repo and setup miniconda\n",
-        "\n",
-        "After runtime is executed, colab will complain that it crashed, but everything is fine, just hit run on the next cell or Runtime-> Run After\n",
-        "\n"
-      ],
-      "metadata": {
-        "id": "RXasLtrYC9uj"
-      }
+        "Clone git repo and setup miniconda\n"
+      ]
     },
     {
       "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "cellView": "form",
+        "id": "LSCxESSsDGVh"
+      },
+      "outputs": [],
       "source": [
         "#@markdown ## Download the stable-diffusion repo from hlky\n",
         "#@markdown And install colab related conda\n",
+        "%cd /content\n",
         "!git clone https://github.com/hlky/stable-diffusion\n",
         "%cd /content/stable-diffusion\n",
-        "!git checkout c0c2a7c0d55561cfb6f42a3681346b9b70749ff1\n",
-        "!pip install -e .\n",
-        "!pip install condacolab\n",
-        "import condacolab\n",
-        "condacolab.install_miniconda()"
-      ],
-      "metadata": {
-        "cellView": "form",
-        "id": "LSCxESSsDGVh"
-      },
-      "execution_count": null,
-      "outputs": []
+        "\n",
+        "#replace condacolab with regular miniconda to avoid the kernel restart\n",
```

Comment:
this comment prob need to be removed as it only makes sense for us and not users who don't know what either of those are

Reply:
Fair enough